Kitodo/src/main/java/de/sub/goobi/forms/BenutzerverwaltungForm.java

```diff
@@ -131,6 +132,20 @@ public String filterAlleStart() {
     }
 
     /**
+     * Return list of users.
+     *
+     * @return list of user groups
+     */
+    public List<UserDTO> getUsersForTasks() {
```

Comment:
Where is the correlation the the task here? You just get allActiveUsers

Reply:
you need this list for popup window with users

Reply:
but why not call it "getActiveUsers"?

Reply:
Because in the future getActiveUsers will contain active users with project and user group titles, and this one will contain only list of users without any additional data - so slim version used only for tasks' popup.